### DISCUSSION TITLE
Play song without starting radio

### DISCUSSION THREAD

**Discussion by Anlaoui** [fast4x/RiMusic#1968] [Q&A] [answered]
Hello! Is it possible, when playing a song from search, to have only this song played and not automatically populate the queue with hours of suggested songs?
Most of the time when I want to listen to a song, I don't want to automatically start a radio based on it.
Maybe there's a setting for this but I didn't find it!

**Comment by fast4x** [comment, marked as answer]
Hi create a feature request because actually there isn't any settings for this.

**Reply by Anlaoui** [reply]
Thanks for the answer, I just did it here :-) https://github.com/fast4x/RiMusic/issues/2165
